Solutions/VMRay Email Threat Defender Connector/Package/createUiDefinition.json

```diff
@@ -6,7 +6,7 @@
     "config": {
       "isWizard": false,
       "basics": {
-        "description": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/vmray.svg\"width=\"75px\"height=\"75px\">\n\n**Important:** _This Azure Sentinel Solution is currently in public preview. This feature is provided without a service level agreement, and it's not recommended for production workloads. Certain features might not be supported or might have constrained capabilities. For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/)._\n\n**Note:** _There may be [known issues](https://aka.ms/sentinelsolutionsknownissues) pertaining to this Solution, please refer to them before installing._\n\nThis connector ingests email data collected by [VMRay Email Threat Defender (ETD)](https://www.vmray.com/products/email-threat-defender/). It requires a connection to a VMRay Platform 4.3.0 (or later) installation that has ETD enabled.\n\nAzure Sentinel Solutions provide a consolidated way to acquire Azure Sentinel content like data connectors, workbooks, analytics, and automations in your workspace with a single deployment step.\n\n**Data Connectors:** 1\n\n[Learn more about Azure Sentinel](https://aka.ms/azuresentinel) | [Learn more about Solutions](https://aka.ms/azuresentinelsolutionsdoc)",
+        "description": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/vmray.svg\"width=\"75px\"height=\"75px\">\n\n**Note:** _There may be [known issues](https://aka.ms/sentinelsolutionsknownissues) pertaining to this Solution, please refer to them before installing._\n\nThe VMRay Email Threat Defender Connector solution provides the capability to ingest email data collected by [VMRay Email Threat Defender (ETD)](https://www.vmray.com/products/email-threat-defender/) into Microsoft Sentinel. It requires a connection to a VMRay Platform 4.3.0 (or later) installation that has ETD enabled. \n\n**Underlying Microsoft Technologies used:**\n\n This solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs: \n\n a. [Azure Monitor HTTP Data Collector API](https://docs.microsoft.com/azure/azure-monitor/logs/data-collector-api) \n\n b. [Azure Functions](https://azure.microsoft.com/services/functions/#overview)\n\n**Data Connectors:** 1\n\n[Learn more about Microsoft Sentinel](https://aka.ms/azuresentinel) | [Learn more about Solutions](https://aka.ms/azuresentinelsolutionsdoc)",
```

Comment:
Hi @v-dvedak, description link redirects into page not available page

![image](https://user-images.githubusercontent.com/104008048/175477398-be032ac8-6265-498a-97d9-5d313d49063e.png)

![image](https://user-images.githubusercontent.com/104008048/175477428-0f630163-f5bb-4140-bdbe-64cad5f5f4e5.png)